src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,77 @@
+import cn from 'classnames';
+import { FilterStatus } from '../../types/FilterStatus';
+
+type Props = {
+  filterStatus: FilterStatus,
+  onFilterStatus: (filterStatus: FilterStatus) => void,
+};
+
+export const TodoFooter: React.FC<Props> = ({
+  filterStatus,
+  onFilterStatus,
+}) => {
+  const handleFilterChanging = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    newStatus: FilterStatus,
+  ) => {
+    event.preventDefault();
+    if (newStatus !== filterStatus) {
+      onFilterStatus(newStatus);
+    }
+  };
+
+  return (
+    <footer className="todoapp__footer">
+      <span className="todo-count">
+        3 items left
+      </span>
+
+      {/* Active filter should have a 'selected' class */}
```

Comment:
Remove all comments. Try it write obvious code that explain all we need without comments)